core/util/jit_util.h

```diff
@@ -13,6 +13,9 @@ inline std::string node_info(const torch::jit::Node* n) {
   std::stringstream ss;
   ss << *n;
   std::string node_info = ss.str();
+  // Nodes in torchscript graph have file name and line numbers commented for every node. Remove that when returning a
+  // node name for easier readability.
+  node_info = node_info.substr(0, node_info.find("#", 0));
```

Comment:
Can we make an alternative to node_info if you want this. I think for certain cases full info is better